java/sage/SFIRTuner.java

```diff
@@ -601,9 +601,21 @@ public void playTuneString(String remoteName, String cmdString, boolean forceSyn
     {
       if (Sage.DBG) System.out.println("Playing IR tune command of " + cmdString);
       if (!ensureRemoteLoaded(remoteName)) return;
+
+      try {
+         int cmdNum = Integer.parseInt(cmdString);
+      } catch (Exception e) {
+         String cmdStringNumeric = cmdString.replaceAll("\\D", ""); // remove all non-digits
+	 cmdString = cmdStringNumeric;
+         if (Sage.DBG) System.out.println("IR tune command must be all digits; converted it to: " + cmdString);
+      }
+
       if (canMacroTune())
       {
-        macroTune(Integer.parseInt(cmdString));
+	try {
+          macroTune(Integer.parseInt(cmdString));
+	}
+        catch (Exception e){}
```

Comment:
Just log an error if there's some other exception here (which of course there shouldn't be, but the native plugin could propagate something up).

Reply:
Done.  My editor got carried away and auto-inserted tabs in place of spaces.